observability/otel/otel-rn.adoc

```diff
@@ -41,7 +44,10 @@ include::snippets/technology-preview.adoc[leveloffset=+1]
 
 This update introduces the following enhancements:
 
-* ???.
+// https://issues.redhat.com/browse/TRACING-3768
+* Collector dashboard for internal collector metrics. The dashboard helps to analyze collector health and performance.
+// https://issues.redhat.com/browse/TRACING-3898
+* The operator is capable of automatically creating RBAC for collector components that require custom RBAC.
```

Comment:
It was before but now we changed how the operator knows if it can do it or not

Reply:
I've moved your comment into https://github.com/openshift/openshift-docs/pull/80135/files#r1724714954.